Cargo.toml

```diff
@@ -8,11 +8,11 @@ license = "Apache-2.0 OR BSD-3-Clause"
 edition = "2018"
 
 [dependencies]
-byteorder = "~1.2.1"
+byteorder = "~1.3.1"
 log = "0.4"
-kvm-bindings = "~0"
-kvm-ioctls = "~0"
+kvm-bindings = { git = "https://github.com/cloud-hypervisor/kvm-bindings", branch = "ch" }
+kvm-ioctls = { git = "https://github.com/cloud-hypervisor/kvm-ioctls", branch = "ch" }
 vfio-bindings = "~0"
 vmm-sys-util = "~0"
 
-vm-memory = { git = "https://github.com/rust-vmm/vm-memory", features = ["backend-mmap"] }
+vm-memory = { version = "0.2.0" , features = ["backend-mmap", "backend-atomic"] }
```

Comment:
How about "version = "^0, >=0.2", please refer https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

Reply:
Yes, it makes sense, thanks :)